Tests/AddStudentPageTests.cs

```diff
@@ -1,12 +1,81 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using NUnit.Framework;
+using WebDriverStudentsPOM.Pages;
 
 namespace WebDriverStudentsPOM.Tests
 {
     public class AddStudentPageTests : BaseTest
     {
+        private AddStudentPage page;
+
+        [SetUp]
+        public void Setup()
+        {
+            this.page = new AddStudentPage(driver);
+        }
+
+        [Test]
+
+        public void Test_TestAddStudentPage_Content()
```

Comment:
I think one of the "Test" should be removed from the method's name, e.g. Test_AddStudentPage_Content. Same for the methods names below. Otherwise, all looks good. :)

Reply:
Well noted. Many thanks.